test/multiverse/suites/activemerchant/Envfile

```diff
@@ -4,8 +4,6 @@
 
 ACTIVEMERCHANT_VERSIONS = [
   [nil, 3.0],
```

Comment:
Oh good catch, I had missed noticing that the min version here was 3.0. I'm not sure why it would be 3.0 though? I just tested running it on 2.6 locally and it worked fine, so i'll update that to 2.5 and see how that does on the CI. 
172d6b3fc3e34ed46154722da0c3b5ea097923bb